tajo-core/pom.xml

```diff
@@ -690,6 +696,37 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>parallel-test</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
Sorry for the confusion. I was test successfully with TAJO-1322, TAJO-1323 on surefire-2.18.1